ci-operator/step-registry/cucushift/installer/check/capability/cucushift-installer-check-capability-commands.sh

```diff
@@ -158,9 +162,13 @@ check_result=0
 echo "------check enabled capabilities-----"
 echo "enabled capability set: ${enabled_capability_set}"
 for cap in $enabled_capability_set; do
-    if [[ "${cap}" == "Build" ]] || [[ "${cap}" == "DeploymentConfig" ]]; then
+    echo "check capability ${cap}"
```

Comment:
From the [output](https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/43585/rehearse-43585-periodic-ci-openshift-openshift-tests-private-release-4.12-multi-nightly-azure-ipi-baselinecaps-vset-additionalcap-arm-f28/1711258461511094272/artifacts/azure-ipi-baselinecaps-vset-additionalcap-arm-f28/cucushift-installer-check-capability/build-log.txt), seem like there are some duplicates, I guess we can do some uniq operations on line 150:
```
enabled_capability_set=($(echo "${enabled_capability_set[@]}" | tr ' ' '\n' | sort -u | tr '\n' ' '))
```